api/app/tests/Behat/features-v2/user-management/org/delete-others.feature

```diff
@@ -0,0 +1,17 @@
+@v2 
+Feature: Org - A professional deputy deletes users from org
+
+    @prof-admin-combined-high-not-started
+    Scenario: A professional deputy with admin permissions can delete users from org
+        Given the user has 'admin' permissions and another user exists within the same organisation
+        When the user I'm interacting with logs in to the frontend of the app
+        And I visit the organisation settings user account page for the logged in user
+        And I attempt to remove an org user 
+        Then the the user should be deleted
```

Comment:
Quite minor but just to be clear about what's happening to the user here - are they deleted from the system or from the org?

Reply:
Good point! The user should be removed from org so I've updated the step